src/vstest.console/CommandLine/TestRunResultAggregator.cs

```diff
@@ -107,13 +107,11 @@ public void Reset()
         /// </summary>
         private void TestRunCompletionHandler(object sender, TestRunCompleteEventArgs e)
         {
-            if (e.TestRunStatistics == null)
+            if (e.TestRunStatistics == null || e.IsCanceled || e.IsAborted)
             {
```

Comment:
For now we are returning exit code `1` for cancel , fail and abort tests. In future we are planning to have different exit code for each  cancel , fail and abort